RecoLocalTracker/SiPixelRecHits/src/SiPixelTemplate.cc

```diff
@@ -2263,237 +2345,309 @@ void SiPixelTemplate::xtemp3d(int i, int j, std::vector<float>& xtemplate)
+//! \param id - (input) index of the template to use
+//! \param cotalpha - (input) the cotangent of the alpha track angle (see CMS IN 2004/014)
+//! \param cotbeta - (input) the cotangent of the beta track angle (see CMS IN 2004/014)
+//! \param locBz - (input) the sign of this quantity is used to determine whether to flip cot(beta)<0 quantities from cot(beta)>0 (FPix only)
+//!                    for Phase 0 FPix IP-related tracks, locBz < 0 for cot(beta) > 0 and locBz > 0 for cot(beta) < 0
+//!                    for Phase 1 FPix IP-related tracks, see next comment
+//! \param locBx - (input) the sign of this quantity is used to determine whether to flip cot(alpha/beta)<0 quantities from cot(alpha/beta)>0 (FPix only)
+//!                    for Phase 1 FPix IP-related tracks, locBx/locBz > 0 for cot(alpha) > 0 and locBx/locBz < 0 for cot(alpha) < 0
+//!                    for Phase 1 FPix IP-related tracks, locBx > 0 for cot(beta) > 0 and locBx < 0 for cot(beta) < 0//!
+//! \param qclus - (input) the cluster charge in electrons
+//! \param pixmax - (output) the maximum pixel charge in electrons (truncation value)
+//! \param sigmay - (output) the estimated y-error for CPEGeneric in microns
+//! \param deltay - (output) the estimated y-bias for CPEGeneric in microns
+//! \param sigmax - (output) the estimated x-error for CPEGeneric in microns
+//! \param deltax - (output) the estimated x-bias for CPEGeneric in microns
+//! \param sy1 - (output) the estimated y-error for 1 single-pixel clusters in microns
+//! \param dy1 - (output) the estimated y-bias for 1 single-pixel clusters in microns
+//! \param sy2 - (output) the estimated y-error for 1 double-pixel clusters in microns
+//! \param dy2 - (output) the estimated y-bias for 1 double-pixel clusters in microns
+//! \param sx1 - (output) the estimated x-error for 1 single-pixel clusters in microns
+//! \param dx1 - (output) the estimated x-bias for 1 single-pixel clusters in microns
+//! \param sx2 - (output) the estimated x-error for 1 double-pixel clusters in microns
+//! \param dx2 - (output) the estimated x-bias for 1 double-pixel clusters in microns
+//! \param lorywidth - (output) the estimated y Lorentz width
+//! \param lorxwidth - (output) the estimated x Lorentz width
+// ************************************************************************************************************
+int SiPixelTemplate::qbin(int id, float cotalpha, float cotbeta, float locBz, float qclus, float& pixmx,
+                          float& sigmay, float& deltay, float& sigmax, float& deltax,float& sy1, float& dy1, float& sy2,
+                          float& dy2, float& sx1, float& dx1, float& sx2, float& dx2) // float& lorywidth, float& lorxwidth)
+{
+   // Interpolate for a new set of track angles
+   
+   // Local variables
+   float locBx; //  lorywidth, lorxwidth;
+   // Local variables
+   locBx = 1.f;
```

Comment:
We agree.  (On an analysis review twiki this would be color-coded as green :)

If we are making other changes, we'll do this.  However, if this is the only change, we maybe should defer this particular (and trivial) fix until the next PR.